src/test/scala/com/github/saurfang/sas/spark/DefaultSourceSuite.scala

```diff
@@ -37,7 +37,7 @@ class DefaultSourceSuite extends FunSuite with Matchers with SharedSparkContext
       s"""
          |CREATE TEMPORARY TABLE datetimeTable
```

Comment:
`TABLE` -> `VIEW`

Reply:
FWIW that change looks entirely reasonable. Should work in Spark 2 and Spark 3.